dev-java/eclipse-ecj/eclipse-ecj-4.15.ebuild

```diff
@@ -12,7 +12,7 @@ DMF="R-${PV}-202003050155"
 
 DESCRIPTION="Eclipse Compiler for Java"
 HOMEPAGE="http://www.eclipse.org/"
-SRC_URI="http://download.eclipse.org/eclipse/downloads/drops4/${DMF}/${MY_PN}src-${PV}.jar"
+SRC_URI="http://archive.eclipse.org/eclipse/downloads/drops4/${DMF}/${MY_PN}src-${PV}.jar"
```

Comment:
While you're at it:
Can use https? And https also for HOMEPAGE?

Reply:
Fixed